Lesson7/organizer7/scheduler/templates/admin_user_info.html

```diff
@@ -0,0 +1,28 @@
+{% extends "./admin_user_list_base.html" %}
+
+{% block title %}
```

Comment:
You do not have the block title in extending the template so it won't be added to the final html page.